test/test_play.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
```

Comment:
One last thing; sorry! It looks like this file needs the standard comment header.